app/src/panels/list/index.ts

```diff
@@ -7,6 +9,19 @@ export default definePanel({
 	description: '$t:panels.list.description',
 	icon: 'list',
 	component: PanelList,
+	query: (options) => {
```

Comment:
We might want to add support for returning `null` in this object, so the parent is aware when to "skip" the panel. This could happen when not all options are (correctly) configured. 

(For example, when `options.collection` isn't set, this will fail, so we should return `null` earlier to prevent it from crashing everything, and then handle that invalid query state on the parent dashboard)

Reply:
Thank you!

Reply:
Either of those work! To me `undefined` means we didn't generate any value (eg something went wrong), while `null` means "we're explicitly telling you there's no result", though that's very subjective